modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,101 @@
+= Migrating Your PostgreSQL Database with a Kogito upgrade
+:flyway_install_url: https://flywaydb.org/download/community
+:flyway_migrate_existing_url: https://flywaydb.org/documentation/learnmore/existing
+:kogito_ddl_script_url: https://repo1.maven.org/maven2/org/kie/kogito/kogito-ddl
```

Comment:
link should be updated or ?

Reply:
The `ddl` one? I think we should if it's a production counter-part and move it to `antora.yaml` and group all the "must be updated downstream links" together.

Reply:
Is this the correct `mvn repository`, it should come from? https://maven.repository.redhat.com/ga/org/kie/kogito/
I can not find the `kogito-ddl` here.

Reply:
@ricardozanini are we shipping any persistence for now ?

Reply:
@cristianonicolai cc

Do we have this DDL for DP4? Otherwise we will need to change the doc.

Reply:
@ricardozanini yeah, persistence has been there for a while now, but I guess we missed including the links to download the ddl zip file. I would say we need to get this included and have the correct link for the productised artefacts.

Reply:
@radtriste @cristianonicolai so how do we productize these DDL zip file?

Reply:
AFAIK these are just maven artefacts that get published in the prod Maven repo

Reply:
FYI I've started a discussion on gchat to discuss how it is done on productization side.

Reply:
As per the discussion, I'll keep the link as it is for now, and it will be fixed in a later release.